pkgs/applications/networking/instant-messengers/webex/default.nix

```diff
@@ -0,0 +1,110 @@
+{ stdenv
+, lib
+, fetchurl
+, dpkg
+, makeWrapper
+, autoPatchelfHook
+, libGL
+, libX11
+, libXdamage
+, libXfixes
+, libXcomposite
+, libXrandr
+, cups
+, libxkbcommon
+, libsecret
+, libxcb
+, xcbutilwm
+, xcbutilrenderutil
+, xcbutilimage
+, xcbutilkeysyms
+, libdrm
+, libxshmfence
+, fontconfig
+, mesa
+, nss
+, wayland
+, udev
+, dbus
+, glib
+, libpulseaudio
+, alsaLib
+, at-spi2-core
+, at-spi2-atk
+, harfbuzz
+}:
+stdenv.mkDerivation {
+  pname = "webex";
+  version = "41.5.0.18815";
+
+  src = fetchurl {
+    url = "https://binaries.webex.com/WebexDesktop-Ubuntu-Official-Package/Webex.deb";
```

Comment:
That needs to be uploaded to web.archive.org or it will break when they update the file.

Reply:
Ohh thank you! I wasn't sure how to deal with that problem.